test/RadworksGovernor.t.sol

```diff
@@ -72,14 +72,13 @@ abstract contract Propose is ProposalTest {
   ) public {
     IERC20 _token = _randomERC20Token(_seed);
     _assumeReceiver(_receiver);
+    _upgradeToBravoGovernor();
     uint256 _timelockTokenBalance = _token.balanceOf(TIMELOCK);
 
     // bound by the number of tokens the timelock currently controls
     _amount = bound(_amount, 0, _timelockTokenBalance);
     uint256 _initialTokenBalance = _token.balanceOf(_receiver);
 
-    _upgradeToBravoGovernor();
```

Comment:
It may better to keep this test as is and `assume` the token does not equal the rad token, and create a new test specifically for the RAD token which subtracts the expected ScopeLift amount from the `_timelockTokenBalance`. In this setup a test will fail if an unexpected amount of tokens is transferred during the upgrade where as I think this will not.

Reply:
Good idea! Thanks!